src/main/java/com/example/hangman/HelloApplication.java

```diff
@@ -0,0 +1,23 @@
+package com.example.hangman;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class HelloApplication extends Application {
```

Comment:
Hey, man. How about we rename this class to reflect the actual name of the app. 
It is named `HelloApplication` by default, when generating the project 😬